### ISSUE TITLE
[BUG] Can't create an EventHub Authorization rule with several access policies

### ISSUE THREAD

**Issue by edrevo** [Azure/azure-libraries-for-java#971]
**Describe the bug**
I want to create an authorization rule for my EventHub namespace that has listen and send access (but no management access)

**To Reproduce**

```scala
az.eventHubNamespaces()
          .authorizationRules()
          .define(AuthorizationRuleName)
          .withExistingNamespaceId(ns.id())
          .withListenAccess()
          .withSendAccess() // This won't compile
          .create()
```

**Expected behavior**
I expect to be able to have both listen and send access on the authorization rule, just like the Azure Portal:

**Screenshots**
![image](https://user-images.githubusercontent.com/1845771/72606951-e65f5c00-391f-11ea-9762-c87142f38222.png)

**Setup (please complete the following information):**
 - OS: Windows 10
 - IDE : IntelliJ 2019
 - "com.microsoft.azure" % "azure" % "1.30.0"

**Information Checklist**
Kindly make sure that you have added all the following information above and checkoff the required fields otherwise we will treat the issuer as an incomplete report
- [x] Bug Description Added
- [x] Repro Steps Added
- [x] Setup information Added

**Comment by yaohaizh** [comment]
Thanks for reporting this issue. We will look into the feature request.

**Comment by weidongxu-microsoft** [comment]
@edrevo

It is a bug. We will schedule a fix.

If you need a quick workaround, you can do this:

```
            EventHubNamespaceAuthorizationRule.DefinitionStages.WithCreate toCreate = azure.eventHubNamespaces()
                    .authorizationRules()
                    .define("rule1")
                        .withExistingNamespaceId("###")
                        .withSendAccess();
            ((EventHubNamespaceAuthorizationRule) toCreate).inner().rights().add(AccessRights.LISTEN);
            toCreate.create();
```

**Comment by weidongxu-microsoft** [comment]
Fixed in 1.31
